operators/intel-n5010-operator/2.5.0/manifests/fpga.intel.com_n5010clusters.yaml

```diff
@@ -0,0 +1,119 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020-2021 Intel Corporation
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.1
+  creationTimestamp: null
+  name: n5010clusters.fpga.intel.com
+spec:
+  group: fpga.intel.com
```

Comment:
@rmr-silicom if you submit new versions of this PR, can you move the CRDs to some other API group please